spec/support/cmless_markdown_tester.rb

```diff
@@ -0,0 +1,59 @@
+require 'byebug'
+
+require 'nokogiri'
+require 'cmless'
+require 'faraday'
+
+class CmlessMarkdownTester
+
+  attr_reader :ng, :path, :markdown, :html
+
+  def initialize(path)
+    @path = path
+    @markdown = File.read(@path)
+    @html = Cmless::Markdowner.instance.render(@markdown)
+    @ng = Nokogiri::HTML(@html)
+  end
+
+  def broken_urls
+    urls.reject{ |url| working_url?(url) }
+  end
+
+  def http_status(url)
+    Faraday.head(url).status rescue nil
+  end
+
+  def working_url?(url)
+    http_status(url).to_i.between? 200, 399
```

Comment:
Agree in principle, but this is mainly for validating links created by Alison and Lynn. I don't think we want to force them to ensure their links are not 301s. Especially if they are committing to `master` without creating PRs.